src/main/java/constants/Options.java

```diff
@@ -9,17 +9,26 @@ public class Options {
     public static final String MANGA_LONG_OPTION = "manga";
     public static final String MANGA_OPTION_DESC = "Manga name";
 
-    public static final String DEADLINE_DATE_OPTION = "d";
+    public static final String DEADLINE_DATE_OPTION = "b";
```

Comment:
Is there any other way we can make it clearer to the user that `b` is for deadline (as in "by"; as discussed offline)?

Reply:
I would say if we specify this (the usage and what it means) in the User Guide it should be fine. But it is possible for me to implement a change to use `--by` instead of `-b` if that is what you mean.

Reply:
Yea making `--by` be the long form of `-b` would be good! The user guide idea sounds good, let's do that too.